web-app/src/pages/Authentication/ConnectWallet.js

```diff
@@ -48,7 +48,7 @@ const ConnectWallet = () => {
             await Wallets[walletType].installWallet();
         }
 
-        setIsLoading(false);
+        setIsLoading(true);
```

Comment:
Why flip this?

Reply:
setIsLoading(true) will let the button disabled but on page load it is enabled.